packages/core/src/components/Tag/components/TagIcon.tsx

```diff
@@ -7,7 +7,6 @@ import { Icon, IconProps } from '../../Icon';
 export interface TagIconProps extends IconProps {}
 
 export const TagIcon: React.FC<TagIconProps> = ({
-  colorScheme = 'neutral',
```

Comment:
If I'm not mistaken, this is in use again.

Reply:
Fixed on https://github.com/DevoInc/genesys-ui/pull/15/commits/9db5d4861bd255a6a60c994c6bd5be3023e82d2b